app/templates/public.hbs

```diff
@@ -9,9 +9,9 @@
             <img src="{{this.model.logoUrl}}" class="logo mr-8" />
           {{/if}}
           <div>
-            <h4 class="event time">{{header-date this.model.startsAt this.model.timezone}}</h4>
```

Comment:
why changed to general

Reply:
We want to move to a single helper, general-date